include/random_walks/uniform_billiard_walk.hpp

```diff
@@ -12,7 +12,7 @@
 #define RANDOM_WALKS_UNIFORM_BILLIARD_WALK_HPP
 
 #include <Eigen/Eigen>
-
+#include <iostream>
```

Comment:
Did you maybe forget to delete it?

Reply:
Yes I'm sorry for this- I deleted it just now